packages/fxa-content-server/app/tests/spec/views/ready.js

```diff
@@ -98,25 +103,24 @@ describe('views/ready', function() {
       );
     }
 
-    it('shows service name if available', function() {
+    it('shows default service name', function() {
       createView(VerificationReasons.SIGN_UP);
-      relier.set('serviceName', 'Firefox Sync');
-
       return view.render().then(function() {
-        var html = view.$('section').text();
-        assert.include(html, 'Firefox Sync');
+        assert.equal(
+          view.$(SIGNIN_COMPLETE.SERVICE_NAME).text(),
+          'You are now ready to use Account Settings'
+        );
       });
     });
 
-    // regression test for #1216
```

Comment:
Doesn't look like we need this test anymore since a `serviceName` will always be provided so I wrote a different test to check for its default value.